js/controls.js

```diff
@@ -583,12 +596,13 @@ const Controls = () =>
      * successful aqcuisition of the resource, in this case, the json file.
      */
     function loadRefAnimation(urlRef) {
-        fetch(urlRef).then((res) => res.json()).then((data) => {
-            loadNewVisualizer(data);
-            updateControls();
+        fetch(urlRef).then((res) => res.json()).then(async (data) => {
+            loadNewVisualizer(data).then(() => {
+                updateControls();
+            });
         }).catch((error) => {
             alert('The specified path to the logfile has returned an error. \
-An example path here is:\n\n :userName/:repoName/branchName/path/to/fileName.json \n\n' + error);
+                An example path here is:\n\n :userName/:repoName/branchName/path/to/fileName.json \n\n' + error);
```

Comment:
The reason why it was not tabbed over was because that shows up in the alert box as a tab of whitespace.

Reply:
Ohhhh. I will fix that